tests/versions/go.yml

```diff
@@ -0,0 +1,5 @@
+GO_AGENT:
+  - 'github;master'
+  - 'github;v0.4.0'
```

Comment:
don't worry about 0.x, we're not going to be maintaining them long-term

Reply:
ok, but let's keep it there to have two versions for the axis tests, when we have a version 1.X.X we would change it.

Reply:
0.x won't work, since the import paths are different. Supporting both will require significant changes. I'll create a 1.x branch soon.

Reply:
in that case, I remove them and keep only master, when we have more version we can add them